images/microOS/config.xml

```diff
@@ -593,120 +140,67 @@
         <package name="live-add-yast-repos"/>
         <!-- Apparently zypper (?) doesn't like having multiple patterns with the same name,
              so avoid using namedCollection at least for basesystem -->
-        <package name="patterns-microos-basesystem"/>
+        <package name="patterns-base-base"/>
         <package name="patterns-base-bootloader"/>
-        <package name="patterns-microos-defaults"/>
         <package name="kernel-default"/>
     </packages>
 
-    <packages type="image" profiles="Ceph-base">
+    <packages type="image" profiles="Ceph-base" patternType="onlyRequired"> 
         <package name="lvm2"/>
         <package name="chrony"/>
         <package name="tuned"/>
-        <package name="python3-pip"/>
-        <package name="python3-aiofiles"/>
-        <package name="python3-requests"/>
+        <package name="python38-pip"/>
+        <package name="python38-aiofiles"/>
+        <package name="python38-requests"/>
         <package name="python3-rados"/>
-        <package name="python3-websockets"/>
+        <package name="python38-websockets"/>
         <archive name="aquarium.tar"/>
     </packages>
 
-    <packages type="image" profiles="kvm-and-xen,kvm-and-xen_x86_64,kvm-and-xen_aarch64,VMware,MS-HyperV,VirtualBox,Pine64,RaspberryPi,RaspberryPi2,Vagrant_x86_64,Vagrant_aarch64">
+    <packages type="image" profiles="Vagrant-x86_64">
         <!-- Needed for initial configuration, except on OpenStack-Cloud, where cloud-init is used -->
         <package name="ignition-dracut"/>
         <package name="combustion"/>
         <package name="growpart-generator"/>
     </packages>
-    <packages type="image" profiles="ContainerHost">
-        <namedCollection name="container_runtime"/>
-        <package name="patterns-containers-container_runtime"/>
-        <namedCollection name="microos_apparmor"/>
```

Comment:
I think we need to include `patterns-base-apparmor`, otherwise we end up with an empty `/sys/kernel/security/apparmor/profiles`.

Reply:
Let me test that later with the apparmor patterns included